lib/matplotlib/axis.py

```diff
@@ -2020,7 +2020,10 @@ def get_tick_space(self):
         # There is a heuristic here that the aspect ratio of tick text
         # is no more than 3:1
         size = tick.label1.get_size() * 3
-        return int(np.floor(length / size))
+        if size > 0:
+            return int(np.floor(length / size))
+        else:
```

Comment:
I know order doesn't matter, I just think from a clarity/maintance point of view the special case should be in the else. 

Though now I'm wondering if there's a way to modify `int(np.floor(length/size))` such that it'll catch the size<=0 case